spec/rspec/core/option_parser_spec.rb

```diff
@@ -394,5 +394,22 @@ module RSpec::Core
       end
     end
 
+    describe '--force-color' do
+      it 'aborts if --no-color was previously set' do
+        parser = Parser.new(%w[--no-color --force-color])
+        expect(parser).to receive(:abort).with(a_string_including('only use one of `--force-color` and `--no-color`'))
+        parser.parse
+      end
+    end
+
+    describe '--no-color' do
+      it 'aborts if --force-color was previously set' do
+        parser = Parser.new(%w[--force-color --no-color])
+        expect(parser).to receive(:abort) do |msg|
+          expect(msg).to include('only use one of --force-color and --no-color')
```

Comment:
Sounds good. I went ahead and changed the pre-existing `:abort`/`use --help` specs to use `a_string_including` too while I was in there, that way the whole file uses the same approach.